website_form_recaptcha/static/src/js/field_recaptcha.js

```diff
@@ -6,28 +6,26 @@ odoo.define('website_form_recaptcha.recaptcha', function(require){
   "use strict";
   
   var snippet_animation = require('web_editor.snippets.animation');
-  var model = require('web.Model');
   
   snippet_animation.registry.form_builder_send = snippet_animation.registry.form_builder_send.extend({
     
     start: function() {
       var self = this;
       this._super();
+      this.$captchas = self.$('.o_website_form_recaptcha');
       $.ajax({
         url: '/website/recaptcha/',
         method: 'GET',
         success: function(data){
           data = JSON.parse(data);
-          var $captchas = self.$target.find('.o_website_form_recaptcha');
-          $captchas.append($(
+          this.$captchas.append($(
```

Comment:
I know this works, but it's better if you explicitly set `var $ = require("$");` at the imports section.